mlflow/sklearn/__init__.py

```diff
@@ -682,6 +693,11 @@ def fetch_logged_data(run_id):
         MAX_ENTITY_KEY_LENGTH,
     )
 
+    if max_tuning_runs is not None and max_tuning_runs < 0:
+        raise ValueError(
```

Comment:
Can we raise `MlflowException` with error code `INVALID_PARAMETER_VALUE` instead?